packages/settings/api.d.ts

```diff
@@ -1,5 +1,8 @@
 import Thenable from "vscode-languageserver";
-export type Settings = CodeAssistSettings & TraceSettings & LoggingSettings;
+export type Settings = CodeAssistSettings &
+  TraceSettings &
+  LoggingSettings &
+  API_Reference;
```

Comment:
The naming of `API_Reference` type does not seem consistent 
with the existing pattern in this code base / file